tests/chez/chez028/ExpressionParser.idr

```diff
@@ -11,45 +11,45 @@ import Data.String.Parser.Expression
 
 table : OperatorTable Nat
 table =
-  [ [Infix (do token "^"; pure (power) ) AssocRight]
-  , [ Infix (do token "*"; pure (*) ) AssocLeft ]
-  , [ Infix (do token "+"; pure (+) ) AssocLeft ]
+  [ [ Infix (token "^" *> pure power ) AssocRight]
+  , [ Infix (token "*" *> pure (*)   ) AssocLeft ]
+  , [ Infix (token "+" *> pure (+)   ) AssocLeft ]
   ]
 
 table' : OperatorTable Integer
 table' =
-  [ [ Infix (do token "*"; pure (*) ) AssocLeft
-    , Infix (do token "/"; pure (div) ) AssocLeft
+  [ [ Infix (token "*" *> pure (*)   ) AssocLeft
+    , Infix (token "/" *> pure (div) ) AssocLeft
     ]
-  , [ Infix (do token "+"; pure (+) ) AssocLeft
-    , Infix (do token "-"; pure (-) ) AssocLeft
+  , [ Infix (token "+" *> pure (+) ) AssocLeft
```

Comment:
May be worth adding `(<$)` to `base` (or `prelude`, wherever `(<$>)` is defined)?
I often reach for it only to find it is missing.